### DISCUSSION TITLE
Refund request

### DISCUSSION THREAD

**Discussion by albertphil** [aclap-dev/video-downloadhelper#1141] [License & Purchase requests]
### What's your issue?

I can't download videos on the site I intended to use it with.
I'd like to request a refund.

### The 10 first digits (only 10) of your license number or email address

2649b32ff6

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
